executor/grant_test.go

```diff
@@ -415,6 +416,16 @@ func (s *testSuite3) TestIssue22721(c *C) {
 	tk.MustExec("GRANT USAGE ON test.xx TO 'sync_ci_data'@'%';")
 }
 
+func (s *testSuite3) TestIssue27867(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("create user issue27867;")
+	c.Assert(tk.Se.Auth(&auth.UserIdentity{Username: "root", Hostname: "localhost"}, nil, nil), IsTrue)
+	err := tk.ExecToErr("grant all on performance_schema.* to issue27867;")
```

Comment:
Can you test on `PERFormanCE_schema` as well? Schema objects are insensitive in TiDB. There might have been a lower prior to executor/grant.go line 493

Reply:
Added.